samples/meetings-attendance-report/csharp/MeetingAttendance/Resources/AttendanceSummaryCard.json

```diff
@@ -0,0 +1,185 @@
+{
+  "type": "AdaptiveCard",
+  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+  "version": "1.4",
+  "body": [
+    {
+      "type": "TextBlock",
+      "size": "Large",
+      "weight": "Bolder",
+      "text": "Summary"
+    },
+    {
+      "type": "ColumnSet",
+      "columns": [
+        {
+          "type": "Column",
+          "items": [
+            {
+              "type": "TextBlock",
+              "text": "Attended Participant",
+              "wrap": true
+            },
+            {
+              "type": "TextBlock",
+              "weight": "Bolder",
+              "text": "${participantCount}",
+              "wrap": true,
+              "size": "Large"
+            }
```

Comment:
If we have multiple people joined the meeting, how are we handling dynamically and showing ?

Reply:
Using $data property of Adaptive card multiple elements can be rendered